packages/jaeger-ui/src/components/DeepDependencies/Header/LayoutSettings/index.test.js

```diff
@@ -66,15 +68,6 @@ describe('LayoutSettings', () => {
     expect(trackDensityChangeSpy).toHaveBeenCalledWith(props.density, newDensity, densityOptions);
   });
```

Comment:
there should be two new tests:
* one that verifies that if a value is found in the localStorage than that value is used
* another that verifies that when changing the selection in the UI the value is recorded in localStorage

Reply:
Got it, will start working on that!

Reply:
Update: I have finished writing the tests, should I raise a PR?